tests/integration/17.SecurityTag.Tests.ps1

```diff
@@ -28,20 +28,38 @@ Describe -Name "SecurityTagAssignment" -Tag "Get" -Fixture {
         #We load the mod and establish connection to NSX Manager here.
 
         #Put any setup tasks in here that are required to perform your tests.  Typical defaults:
+        Write-Verbose -Verbose "Performing setup tasks for SecurityTag tests"
```

Comment:
really need add verbose ?

Reply:
Greets-  _Need_ verbose?  Possibly not. The reason for `Write-Verbose -Verbose` here, though, is to deliver a verbose message that had previously been delivered via `Write-Warning` (a message that was not a warning).  And, by "previously delivered", I mean that was in the test code from this module off of which I based this new test file.

So, if we want a verbose message, then, yes, we keep this.  If not, of course, let's whack it.